action/protocol/staking/vote_bucket.go

```diff
@@ -108,7 +173,7 @@ func stakingGetTotalCount(sr protocol.StateReader) (uint64, error) {
 	return tc.count, err
 }
 
-func stakingGetBucket(sr protocol.StateReader, name CandName, index uint64) (*VoteBucket, error) {
+func stakingGetBucket(sr protocol.StateReader, name hash.Hash160, index uint64) (*VoteBucket, error) {
```

Comment:
address.Address and same for the rest of this PR

Reply:
changed